tests/xoto3/dynamodb/utils/expressions_test.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from xoto3.dynamodb.utils.expressions import add_variables_to_expression
+
+
+def test_add_variables_to_expression():
+    query_dict = dict()
+    variables = dict(deletedAt="2020-01-01T00:00:00.000000Z")
+
+    add_variables_to_expression(query_dict, variables)
+
+    assert query_dict["ExpressionAttributeNames"] == {"#deletedAt": "deletedAt"}
+    assert query_dict["ExpressionAttributeValues"] == {":deletedAt": "2020-01-01T00:00:00.000000Z"}
+
+
+def test_add_variables_to_expression_with_bad_attribute_name():
+    query_dict = dict()
+    variables = dict(thingy="THINGY", deleted__At="2020-02-02T00:00:00.000000Z")
+
+    with pytest.raises(ValueError, match="Attribute name contains invalid characters"):
```

Comment:
Could this be?
```suggestion
    with pytest.raises(ValueError, match="Attribute 'deleted__At' contains invalid characters"):
```

Reply:
Code already emitted it after the exception, but the test didn't reflect it...